### PR TITLE
Add auto theme mode

### PR DESCRIPTION
**Pull request by kuenzign**
Add another mode which will automatically select the theme based on the user's OS or browser theme preferences.

### CODE REVIEW

File: directory-listing/table-automode.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import 'table.css';
+@media (prefers-color-scheme: dark) {
+  @import 'darkmode.css';
```
Comment (notlmn) [inline directory-listing/table-automode.css]: Why not import `table-darkmode.css` here?

File: directory-listing/grid-automode.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@import 'grid.css';
+@media (prefers-color-scheme: dark) {
+  @import 'darkmode.css';
```
Comment (notlmn) [inline directory-listing/grid-automode.css]: Import `grid-darkmode.css` here. Makes logical sense with comment below.

**Review by notlmn — CHANGES_REQUESTED** [review]
Thanks for the CL

File: directory-listing/table-automode.css (hunk 1 above)
Comment (kuenzign) [reply directory-listing/table-automode.css]: Doing so would import `table.css` twice.  This probably isn't a major issue and I think your suggestion makes more sense from a maintenance perspective, but I think it should be noted.

**Review by notlmn — APPROVED** [review]

**Pull request merged**